back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/exporters/idea_xlsx_form_exporter.rb

```diff
@@ -71,7 +71,7 @@ def importer_data
         }
       end
       options = @form_fields.map do |field|
-        field.options.map do |option|
+        field.ordered_transformed_options.map do |option|
           {
```

Comment:
Where does this `ordered_transformed_options` come from? Was it an existing function and we just had a bug here? :)